src/components/inputs/content-switcher/content-switcher.component.tsx

```diff
@@ -20,7 +20,7 @@ const ContentSwitcher: React.FC<FormFieldProps> = ({ question, onChange, handler
 
   useEffect(() => {
     if (!isEmpty(previousValue)) {
-      const { value } = previousValue;
+      const value = previousValue;
```

Comment:
Do we need the `value` state?

Reply:
thanks @samuelmale ,,,going to raise a fix for this

Reply:
> Do we need the `value` state?

@samuelmale added the[ fix ](https://github.com/openmrs/openmrs-form-engine-lib/pull/306)